README.md

```diff
@@ -86,315 +54,86 @@ By focusing on the re-use of standard protocols, formats and vocabularies, and
-to provide universal usernames/IDs for Solid apps, and to refer to unique Agents
-(people, organizations, devices). (See the [Authentication and User
-Profiles](#authentication-and-user-profiles) section below, for more details).
-
-This provides several benefits:
-
-* The convenience of a persistent, user-controlled reusable sign-in
-* Can be as descriptive or anonymous as the user wants
-* Users have the option of creating multiple identities/logins,
-    to protect separation of concerns (work-life, etc)
-
-* Persistent de-centralized identities mean that a user has to assemble their
-    social graph (friends/subscription list) only *once*, and then re-use it on
-    many social networks. Prevents the "ghost town" network effect that
-    prevents the formation of new social apps.
-
-* An important step towards building a Web of Trust
-
-#### REST API for resources
-
-Solid uses the [Linked Data Platform (LDP)](http://www.w3.org/TR/ldp/) standard
-(see also [LDP Primer](http://www.w3.org/TR/ldp-primer/)) extensively, as a
-standard way of reading and writing generic Linked Data resources.
-See the **[Solid API](#solid-api)** section below, for API listings and examples.
-
-## Linked Data
+### Protocols and APIs
 
-> Linked Data is simply about using the Web to create typed links between data
-  from different sources. These may be as diverse as databases maintained by two
-  organizations in different geographical locations, or simply heterogeneous
-  systems within one organization that, historically, have not easily
-  interoperated at the data level. Technically, Linked Data refers to data
-  published on the Web in such a way that it is machine-readable, its meaning is
-  explicitly defined, it is linked to other external data sets, and can in turn be
-  linked to from external data sets.
-  *(from [Linked Data - Bizer, Heath and Berners-Lee
-  (2009)](http://tomheath.com/papers/bizer-heath-berners-lee-ijswis-linked-data.pdf))*
+We mentioned that applications that read, create and manipulate data are completely
+decoupled from where the data is stored. This means that applications and datastores
+(servers) need a common protocol to speak to each other.
 
-The Solid project is based on
-[Linked Data](https://en.wikipedia.org/wiki/Linked_data)
-principles. These can be roughly summarized as:
+Datastores use a RESTful API, and store data as RDF. Every *resource* in the data has
+a URL, whether this is a document, an image, a person, an abstract concept, or a
+relationship between two resources. Applications can use HTTP to create, read, update
+and delete resources in a datastore. Resources can be organised hierarchically in
+*containers*, which can be thought of like folders/directories.
 
-* HTTP URLs make fantastic globally unique IDs (and because they're HTTP, people and
-  machines can easily look up/dereference those links)
+Simply, to read the attributes of any resource, an application does a GET request on
```

Comment:
> Simply, to read the attributes of any resource

I'm not sure about this phrasing. (Especially since we removed the 'Intro to RDF' section, which likened RDF to Entity-Attribute-Value models).
Maybe something like `to read the graph representation of a resource`?